tests/components/device_tracker/test_owntracks.py

```diff
@@ -17,13 +17,15 @@
 
 LOCATION_TOPIC = "owntracks/{}/{}".format(USER, DEVICE)
 EVENT_TOPIC = "owntracks/{}/{}/event".format(USER, DEVICE)
+WAYPOINT_TOPIC = 'owntracks/{}/{}/waypoint'.format(USER, DEVICE)
```

Comment:
The one in the owntracks component is for subscription which catches all devices of a user. In the test, we need a topic to publish to that is specific to a (device, user).